### ISSUE TITLE
Show pictures taken previous years on this day

### ISSUE THREAD

**Issue by OlavStornes** [nextcloud/photos#179]
I personally find it entertaining to know what i was doing on the current day in past years, ie. Browsing through my images from 6.feb 2018, 2017, 2016 and so on. I'm aware that snapchat has a feature like this with saved images, but it is limited to snaps that you upload to their servers.

Wouldn't it be a nice addition to Nextcloud photos for a function like this as well?

I'm assuming this would be possible by leveraging the EXIF-data if it's present, with a fallback to the modified date (or only focus on the modified date)

**Comment by jancborchardt** [comment]
Yes, totally agree with this, it would be a very nice and playful addition.

We could simply have a section "Memories" (or however we want to call it) which shows sections for the years, and e.g. one row of pictures for that day.

**Comment by galandilias** [comment]
With recent developments of Nextcloud this functionality could be incorporated into the dashboard as a plugin but also available in the Photos app itself. This is a feature I am really jelous for when I compare to e.g. Google Photos :)

I am totally for it and looking forward to have it on the short term roadmap :)

**Comment by rjberm** [comment]
yes I second this idea. Perhaps even an app would be useful. Does anyone know if one exists?

**Comment by rscata** [comment]
yes, it would be very useful

**Comment by kamilkobak** [comment]
This would be a very useful feature to go from Googe Photos to Nextcloud.

**Comment by dmcallejo** [comment]
I've been looking for this feature for years! This should be implemented and surely it won't miss pictures like Amazon Photos.

**Comment by shankar005** [comment]
Nextcloud will be even more great if this feature gets implemented,I have been tracking this for so long..when can I expect the release.kindly reply, I am very much excited to use this feature
